### PR TITLE
Reset password validation

### PR DESCRIPTION
**Pull request by baarkerlounger**

### CODE REVIEW

File: config/webpacker.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ default: &default
   cache_manifest: false
 
   # Extract and emit a css file
-  extract_css: true
+  extract_css: false
```
Comment (paulrobertlloyd) [inline config/webpacker.yml:19]: Was this change intentional?

Comment (baarkerlounger) [reply config/webpacker.yml:19]: Yeah, it causes issues for rspec (think you need to compile assets before running tests?) so wanted to revert this until checking it properly. Left the `inline: false` for dev setting though.

File: app/views/devise/passwords/new.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">Reset password</h1>
-      <%= render "devise/shared/error_messages", resource: resource %>
+      <%= f.govuk_error_summary %>
```
Comment (paulrobertlloyd) [inline app/views/devise/passwords/new.html.erb]: Error summary should appear before the page title.

File: config/webpacker.yml (hunk 1 above)
Comment (paulrobertlloyd) [reply config/webpacker.yml:19]: 👍

**Review by paulrobertlloyd — APPROVED** [review]
